src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts

```diff
@@ -0,0 +1,174 @@
+  13. Click Done
+  14. Under Action group, select Add action groups and complete creation process or
+  select appropriate action group
+  15. Under Alert rule details, enter Alert rule name and Description
+  16. Select appropriate resource group to save the alert to
+  17. Check Enable alert rule upon creation checkbox
+  18. Click Create alert rule
+  
+  Use the below command to create an Activity Log Alert for Delete policy assignment
+  
+    az account get-access-token --query "{subscription:subscription,accessToken:accessToken}" --out tsv | xargs -L1
+    bash -c 'curl -X PUT -H "Authorization: Bearer $1" -H "Content-Type:application/json" https://management.azure.com/subscriptions/$0/resourceGroups/<Resource_Group_ToCreate_Alert_In>/providers/microsoft.insights/activityLogAlerts/<Unique_Alert_Name>?api-version=2017-04-01 -d@"input.json"'
+  
+  Where input.json contains the Request body JSON data as mentioned below.
+  
+    {
+      "location": "Global",
+      "tags": {},
+      "properties": {
+      "scopes": [
+        "/subscriptions/<Subscription_ID>"
+      ],
+      "enabled": true,
+      "condition": {
+        "allOf": [
+        {
+          "containsAny": null,
+          "equals": "Administrative",
+          "field": "category"
+        },
+        {
+          "containsAny": null,
+          "equals": "Microsoft.Network/networkSecurityGroups/write",
+          "field": "operationName"
+        }
+        ]
+      },
+      "actions": {
+        "actionGroups": [
+        {
+          "actionGroupId": "/subscriptions/<Subscription_ID>/resourceGroups/<Resource_Group_For_Alert_Group>/providers/microsoft.insights/actionGroups/<Alert_Group>",
+          "webhookProperties": null
+        }
+        ]
+      },
+      }
+    }
+
+  Configurable Parameters for command line:
+
+    <Resource_Group_To Create_Alert_In>
+    <Unique_Alert_Name>
+
+  Configurable Parameters for input.json:
+
+    <Subscription_ID> in scopes
+    <Subscription_ID> in actionGroupId
+    <Resource_Group_For_Alert_Group> in actionGroupId
+    <Alert_Group> in actionGroupId`,
+  
+  references: [
+    'https://azure.microsoft.com/en-us/updates/classic-alerting-monitoring-retirement',
+    'https://docs.microsoft.com/en-in/azure/azure-monitor/platform/alerts-activity-log',
+    'https://docs.microsoft.com/en-in/rest/api/monitor/activitylogalerts/createorupdate',
+    'https://docs.microsoft.com/en-in/rest/api/monitor/activitylogalerts/listbysubscriptionid',
+    'https://docs.microsoft.com/en-us/azure/security/benchmarks/security-controls-v2-logging-threat-detection#lt-4-enable-logging-for-azure-resources',
+  ],  
+  gql: `{
+    queryazureResourceGroup {
+      id
+      __typename
+      activityLogAlerts {
+        region
+        enabled
+        condition {
+          allOf {
+            field
+            equals
+          }
+        }
+      }
+    }
+  }`,
+  resource: 'queryazureResourceGroup[*]',
```

Comment:
@aacotroneo is this a rule that will need to be rewritten once we have `auzureSubscription`?

Reply:
yes